packages/amplication-notifications-service/src/health/base/health.controller.base.ts

```diff
@@ -0,0 +1,19 @@
+import { Get, HttpStatus, Res } from '@nestjs/common';
+import { Response } from 'express';
+import { HealthService } from '../health.service';
+
+export class HealthControllerBase {
```

Comment:
I believe we can safely remove these base classes. I don't see any value for this in a health check controller other than adding more files to the app ;)